addons/website_slides/static/src/js/activity.js

```diff
@@ -42,4 +42,45 @@ function applyInclude(Activity) {
 
 applyInclude(KanbanActivity);
 
-});
\ No newline at end of file
+});
+
+odoo.define('website_slides/static/src/js/activity.js', function (require) {
```

Comment:
Use expected name for master, so `website_slides/static/components/activity/activity.js`

Reply:
Done!